### PR TITLE
Workato announcement

### PR DESCRIPTION
**Pull request by andreia-virmond**
# About

This is a post about...

# Release

Is this post for a specific release? If so which one?

# Blog image idea

The design team will work on a design for the blog image. Add any ideas you have about the graphic image here.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2021-09/workato-connector/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+[Workato](https://www.workato.com/) is an enterprise integration Platform-as-a-Service (iPaaS) that enables you to build workflow automation. You construct workflows through a low-code/no-code visual designer, supporting hundreds of different platforms and systems. Built in Ruby, Workato connectors are defined as a JSON configuration and registered through the Workato development environment. 
+
+We know IT teams still play a significant role in [workflow automations](https://www.workato.com/the-connector/work-automation-index/), but product teams and business operations teams are becoming more involved. This requires quick implementations, which can be optimized with Workato's low-code interface. 
+
+The Workato connector for Octopus Deploy lets you to quickly integrate Octopus Deploy with your workflows. It supports performing numerous actions with Octopus Deploy, including:
+
+- Creating a release
+- Creating resources such as accounts, channels, deployments, environments, projects
+- Deploying a release
+- Getting a deployment process
```
Comment (TeganAli) [inline blog/2021-09/workato-connector/index.md:26]: @robpearson I don't love the word "getting" - would "fetching" be appropriate here or would that alter the meaning?

Comment (robpearson) [reply blog/2021-09/workato-connector/index.md:26]: I think "fetching" or "retrieving" are fine. Fetch is a simpler word.

File: blog/2021-09/workato-connector/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- Getting various resources from Octopus Deploy (i.e. deployment targets, feeds, runbooks, teams, tenants)
```
Comment (TeganAli) [inline blog/2021-09/workato-connector/index.md:27]:
```suggestion
- Fetching various resources from Octopus Deploy (i.e. deployment targets, feeds, runbooks, teams, tenants)
```

File: blog/2021-09/workato-connector/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  - Product
+---
+
+A [Workato connector for Octopus Deploy](https://www.workato.com/integrations/community/octopus-deploy) is now available. Our support for Workato will facilitate a wide variety of workflow scenarios for Octopus Deploy customers.
+
+[Workato](https://www.workato.com/) is an enterprise integration Platform-as-a-Service (iPaaS) that enables you to build workflow automation. You construct workflows through a low-code/no-code visual designer, supporting hundreds of different platforms and systems. Built in Ruby, Workato connectors are defined as a JSON configuration and registered through the Workato development environment. 
+
+We know IT teams still play a significant role in [workflow automations](https://www.workato.com/the-connector/work-automation-index/), but product teams and business operations teams are becoming more involved. This requires quick implementations, which can be optimized with Workato's low-code interface. 
+
+The Workato connector for Octopus Deploy lets you to quickly integrate Octopus Deploy with your workflows. It supports performing numerous actions with Octopus Deploy, including:
```
Comment (TeganAli) [inline blog/2021-09/workato-connector/index.md:21]:
```suggestion
The Workato connector for Octopus Deploy lets you quickly integrate Octopus Deploy with your workflows. It supports performing numerous actions with Octopus Deploy, including:
```

Comment (TeganAli) [inline blog/2021-09/workato-connector/index.md]:
```suggestion
The Workato connector for Octopus Deploy lets you to quickly integrate Octopus with your workflows. It supports performing numerous actions with Octopus Deploy, including:
```

File: blog/2021-09/workato-connector/index.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  - Product
+---
+
+A [Workato connector for Octopus Deploy](https://www.workato.com/integrations/community/octopus-deploy) is now available. Our support for Workato will facilitate a wide variety of workflow scenarios for Octopus Deploy customers.
+
+[Workato](https://www.workato.com/) is an enterprise integration Platform-as-a-Service (iPaaS) that enables you to build workflow automation. You construct workflows through a low-code/no-code visual designer, supporting hundreds of different platforms and systems. Built in Ruby, Workato connectors are defined as a JSON configuration and registered through the Workato development environment. 
+
+We know IT teams still play a significant role in [workflow automations](https://www.workato.com/the-connector/work-automation-index/), but product teams and business operations teams are becoming more involved. This requires quick implementations, which can be optimized with Workato's low-code interface. 
+
+The Workato connector for Octopus Deploy lets you quickly integrate Octopus Deploy with your workflows. It supports performing numerous actions with Octopus Deploy, including:
```
Comment (TeganAli) [inline blog/2021-09/workato-connector/index.md:21]:
```suggestion
The Workato connector for Octopus Deploy lets you quickly integrate Octopus with your workflows. It supports performing numerous actions with Octopus Deploy, including:
```

**Pull request merged**